src/XrdTls/XrdTlsContext.cc

```diff
@@ -366,9 +378,11 @@ const char *sslCiphers = "ALL:!LOW:!EXP:!MD5:!MD2";
 
 XrdSysMutex            ctxMutex;
 #if __cplusplus >= 201103L
-std::atomic<bool>      initDone( false );
+std::atomic<int>      initTlsDone( 0 );
```

Comment:
So, what is wrong with using an atomic bool?

Reply:
Ah, that was because I was changing that area because to not lose the debug initialisation even if Init() had already been called from elsewhere, and I noticed that our AtomicGet may be defined "__sync_fetch_and_or" and on the gcc docs here: https://gcc.gnu.org/onlinedocs/gcc/_005f_005fsync-Builtins.html it mentioned "The object pointed to by the first argument must be of integer or pointer type. It must not be a boolean type."; so I changed it.

Reply:
See my comment on the underlying issue here.